CMakeLists.txt

```diff
@@ -340,11 +330,6 @@ function(RELATIVE_PROTOBUF_GENERATE_CPP SRCS)
 
     set(OUTPUT_PB_SRC "${OUTPUT_PROTO_DIR}/${GENERATED_FILE_WE}.pb.cc")
     set(GENERATED_PROTO "${OUTPUT_PROTO_DIR}/${GENERATED_FILE_WE}.proto")
-    if(NOT (ONNX_NAMESPACE STREQUAL "onnx"))
-      # We need this dummy header generated by gen_proto.py when ONNX_NAMESPACE
```

Comment:
So this doesn’t apply?

Reply:
No need, it builds with another NAMESPACE